vnavmesh/NavmeshManager.cs

```diff
@@ -9,6 +9,7 @@ namespace Navmesh;
 // manager that loads navmesh matching current zone
 public class NavmeshManager : IDisposable
 {
+    public bool AutoMesh = true;
```

Comment:
What is the idea behind this? Is it just to skip automatic load when changing zones? In that case I presume we'd still want to process pending tasks and clear old navmesh on zone changes.

Reply:
Ya this is just for certain situations you may not want it auto loading mesh ilwhen it's not being used, for example my plug8n auto duty only needs the meshes when im dungeons and AitoGsther only needs then when it's running. So having a way to shut off auto generation is handy